### PR TITLE
Ansible playbooks #21977 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/21977 (see description for link to the rendered output)
bsc#1213077

# Target branches

Which documentation version does this PR apply to?

- Master (Default)

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I'd appreciate your review!  See description above for rendered output.

File: modules/administration/pages/ansible-integration.adoc [hunk 1]
```diff
@@ -18,20 +18,27 @@ Supported features are:
 * discovery of playbooks
 * execution of playbooks
 
+For more information:
+
+* The inventory is sorted list of managed Ansible nodes.  For more information about organizing an inventory, see https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html.
```
Comment (jcayouette) [inline modules/administration/pages/ansible-integration.adoc:23]: The inventory is a sorted list

File: modules/administration/pages/ansible-setup-control-node.adoc [hunk 2]
```diff
@@ -66,10 +65,10 @@ For executing remediations you need to install the SCAP security guide package o
 
 
 [[configure-ansible-inventory-files]]
-== Creating Ansible inventory files
+== Create Ansible inventory files
 
 Ansible Integration tools deploy a playbook as an inventory file.
-Therefore, you should create one inventory file for each distinct OS listed in _Table 1_.
+Therefore, you should create one inventory file for each operating system listed in _Table 1_.
```
Comment (jcayouette) [inline modules/administration/pages/ansible-setup-control-node.adoc:71]: ~~Therefore, you should~~ Create one inventory file for each operating system listed in _Table 1_.

Comment (keichwa) [reply modules/administration/pages/ansible-setup-control-node.adoc]: thanks, Joseph!

**Pull request merged**